Documentation/designs/prometheus-operator-cli.md

```diff
@@ -0,0 +1,61 @@
+# Prometheus Operator CLI
+
+## Summary
+
+This document proposes a new CLI tool to manage Prometheus Operator resources, allowing users to create, update, and delete Prometheus Operator resources using a simple CLI interface as well as
+troubleshoot and debug Prometheus Operator resources.
+
+## Background
+
+During KubeCon Paris 2024, we received feedback from users that managing Prometheus Operator resources is difficult and error-prone. Users reported that they often encounter issues when creating, updating, and deleting Prometheus Operator resources, and that troubleshooting and debugging Prometheus Operator resources is challenging, below you can find some of the issues that users reported:
+
+### Why my Prometheus resource is not being created?
+
+People were struggling to troubleshoot why their Prometheus resources were not being created. They were not sure if the issue was with the Prometheus Operator or with the Prometheus resource itself.
+After long investigation, they found out that the Prometheus Operator were only watching the namespace where the Prometheus Operator was deployed, and the Prometheus resource was being created in a different namespace.
```

Comment:
If I remember correctly, the problem was not about creating Prometheus resources but correctly configuring them. Creating just a single "Prometheus" resource is fairly simple, but also setting up the right permissions, attaching service accounts, troubleshooting target discovery was not straight forward